src/UsgsAstroSarSensorModel.cpp

```diff
@@ -521,7 +521,6 @@ csm::EcefCoord UsgsAstroSarSensorModel::imageToGround(
   // Compute the spacecraft position in the new coordinate system
   //   The cross-track unit vector is orthogonal to the position so we ignore it
   double nadirComp = dot(spacecraftPosition, tHat);
-  double inTrackComp = dot(spacecraftPosition, vHat);
```

Comment:
@jessemapel Just want to double check that this wasn't needed in the calculation somewhere? It's not used right now.

Reply:
Nope I just rechecked Randy's notes and it's not used.